generator/generator_tool/generator_tool.cpp

```diff
@@ -43,6 +43,7 @@
 
 #include <cstdlib>
 #include <string>
+#include <routing/cross_mwm_ids.hpp>
```

Comment:
Why angle braces and after standard headers?

Reply:
Fixed.